### PR TITLE
tools-2661 change module path to github.com/aerospike/asconfig

### PR DESCRIPTION
**Pull request by dwelch-spike**

### CODE REVIEW

**Comment by dwelch-spike** [comment]
The purpose of these changes is to make getting sub packages of asconfig easier.
like `go get http://github.com/aerospike/asconfig/asconf` .

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ For more information and usage examples see the [Aerospike Configuration Tool do
 Build asconfig using the included makefile and display usage information.
 
 ```shell
-git clone https://github.com/aerospike/asconfig.git
+git clone https://github.com/github.com/aerospike/asconfig.git
```
Comment (jdogmcsteezy) [inline README.md:15]:
```suggestion
git clone https://github.com/github.com/aerospike/asconfig.git
```

This does not look right

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The built binary is available at bin/asconfig.
 You can also build asconfig using `go build`.
 
 ```shell
-git clone https://github.com/aerospike/asconfig.git
+git clone https://github.com/github.com/aerospike/asconfig.git
```
Comment (jdogmcsteezy) [inline README.md:26]: same

File: .github/workflows/draft-release.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   # workflows to pull release artifacts from
   snyk-test:
-    uses: aerospike/asconfig/.github/workflows/snyk-test.yml@main
+    uses: github.com/aerospike/asconfig/.github/workflows/snyk-test.yml@main
```
Comment (jdogmcsteezy) [inline .github/workflows/draft-release.yml:11]: I don't think these need to be changed

**Review by jdogmcsteezy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dwelch-spike) [reply README.md]: Ooops, replace errors, good catch!

File: README.md (hunk 2 above)
Comment (dwelch-spike) [reply README.md]: changed

File: .github/workflows/draft-release.yml (hunk 3 above)
Comment (dwelch-spike) [reply .github/workflows/draft-release.yml]: replace error, changed

**Review by jdogmcsteezy — APPROVED** [review]
LGTM

**Pull request merged**